index.md

```diff
@@ -15,6 +15,17 @@ tensors. A particular focus of Zarr is to provide support for storage
 using distributed systems like cloud object stores, and to enable
 efficient I/O for parallel computing applications.
 
+![arabidopsis](/images/arabidopsis.png "arabidopsis")
+<center>Image by <a href="https://www.openmicroscopy.org/2020/11/04/zarr-data.html">OME</a></center>
+
+
+Have beautiful data in Zarr? Show us on Twitter and mention [@zarr_dev](https://twitter.com/zarr_dev) to get featured!
```

Comment:
We might want to additionally propose a hash tag. Something like `#zarrdata` or `#beautifulzarr` or ... 🤔

Reply:
Added a tweet intent. Check [here](https://twitter.com/intent/tweet?via=zarr_dev&text=I%20created%20this%20beautiful%20piece%20using%20Zarr%21&hashtags=zarrdata%2Cbeautifulzarr%2Czarr).